### PR TITLE
Hotfix for sample runs

### PR DESCRIPTION
**Pull request by YohannDudouit**
This PR fixes `ex6` and `ex6p` to use deterministic backends of libCEED in the sample runs, and fixes `nurbs_ex11p` to print messages only once.

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#1360](https://github.com/mfem/mfem/pull/1360) | @YohannDudouit | @tzanio | @tzanio + @camierjs | 03/18/20 | 03/18/20 | 03/19/20 | |

### CODE REVIEW

File: examples/ex6.cpp [hunk 1]
```diff
@@ -20,7 +20,8 @@
 //               ex6 -pa -d occa-cuda
 //               ex6 -pa -d raja-omp
 //               ex6 -pa -d ceed-cpu
-//               ex6 -pa -d ceed-cuda
+//               * ex6 -pa -d ceed-cuda
+//               ex6 -pa -d ceed-cuda:/gpu/cuda/ref
```
Comment (tzanio) [inline examples/ex6.cpp]: Is this the best backend to use, what about `/gpu/cuda/shared`?

Comment (YohannDudouit) [reply examples/ex6.cpp]: Good point, thank you.

**Comment by jandrej** [comment]
I hijacked the PR and added the hotfix for #1348.

**Comment by camierjs** [comment]
Shouldn't we switch examples `1` & `1p` to use the `shared` CEED GPU backend also?
Even if we don't see differences now with the tests, we might if we add some new features.

**Comment by tzanio** [comment]
> Shouldn't we switch examples `1` & `1p` to use the `shared` CEED GPU backend also?
> Even if we don't see differences now with the tests, we might if we add some new features.

I am OK either way,  but it may be better to wait until we start seeing the differences (in particular we will know what change exposes them)

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Comment by camierjs** [comment]
Works fine on `pascal` too, thanks @YohannDudouit for the fix!

**Pull request merged**